File: frontend/src/lib/seller-branding.ts
export const SELLER_DISPLAY_NAME = 'PARLOUR HOUSE';

const LEGACY_SELLER_NAMES = new Set(['Colors Queen', 'Mahaganpati', 'Pranjay']);

export function displaySellerName(name?: string | null): string {
    if (!name || LEGACY_SELLER_NAMES.has(name)) {
        return SELLER_DISPLAY_NAME;
    }
    return name;
}

File: backend/app/core/seller_branding.py
"""Shared seller display name for admin / imported catalog products."""

SELLER_DISPLAY_NAME = "PARLOUR HOUSE"
LEGACY_SELLER_NAMES = frozenset({"Colors Queen", "Mahaganpati", "Pranjay"})


def normalize_seller_name(name: str | None) -> str:
    if not name or name in LEGACY_SELLER_NAMES:
        return SELLER_DISPLAY_NAME
    return name
